lib/Onnxifi/Base.h

```diff
@@ -81,6 +94,7 @@ class Graph {
 
 private:
   BackendPtr backendPtr_;
+  Function *function_;
```

Comment:
Just curious, what does `Function` in glow mean?

Reply:
It represents the compute graph: variables + actual operations. It's backend independent and needs to be compiled down to a specific backend.
That should be done on graphInit as well, I'll add that later.